rust/cli/tests/cli_test.rs

```diff
@@ -569,3 +569,60 @@ async fn test_redshift_snapshots() {
         assert_eq!(current_date.date(), updated_quary_valid_from.date());
     }
 }
+
+/// This test simulates a workflow where a model is built twice in a Postgres database.
+/// In Postgres DROP TABLE IF EXISTS requires a CASCADE if there are view dependencies
+/// Error we are trying to avoid: cannot drop view transform.model_1 because other objects depend on it
+/// replace with materialized_view field in the schame to: view, materialized_view & table to test difference scenarios
+#[tokio::test]
+#[ignore]
+async fn test_postgres_build_model_twice() {
+    // Setup
+    let name = "quary";
+    let temp_dir = tempdir().unwrap();
+    let project_dir = temp_dir.path();
+
```

Comment:
With `testcontainers`, we have the infrastructure not to ignore this, I think we should actually implement this. I definitely think we should implement this to make it work.